pythran/pythonic/include/numpy/random/generator.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef PYTHONIC_INCLUDE_NUMPY_RANDOM_GENERATOR_HPP
+#define PYTHONIC_INCLUDE_NUMPY_RANDOM_GENERATOR_HPP
+
+#include <random>
+
+namespace pythonic
+{
+  namespace numpy
+  {
+    namespace random
+    {
+      namespace details
+      {
+        // Found by reading Numpy source code RNG values
+        // in (random/mtrand/randomkit.c),
+        // but the random generator type is not explicitly mentioned.
+        using default_numpy_generator_t = std::mt19937;
+        default_numpy_generator_t generator;
```

Comment:
Hum we probably don't have too, but it doesn't cost much. And it might avoid bad surprises for numpy users, no?
And according to cppreference.com
> default_random_engine 	implementation-defined

So I guess it wouldn't hurt to impose the same random generator whatever the platform. What do you think?